biopet-utils/pom.xml

```diff
@@ -31,6 +31,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>com.typesafe.play</groupId>
```

Comment:
Isn't the play framework a little too big for just making HTTP calls? AFAIK there are smaller libraries (e.g. [Dispatch](https://dispatchhttp.org/Dispatch.html))